.travis.yml

```diff
@@ -54,221 +43,166 @@ after_script:
 jobs:
   allow_failures:
     - php: nightly
-    - env: DB=pgsql POSTGRESQL_VERSION=11.0
-
-  exclude:
-    - php: 7.1
-      env: DB=sqlite
-    - php: 7.1
-      env: DB=mysql
-    - php: 7.1
-      env: DB=mysqli
-    - php: 7.2
-      env: DB=sqlite
-    - php: 7.2
-      env: DB=mysql
-    - php: 7.2
-      env: DB=mysqli
-    - php: 7.3
-      env: DB=mysql
-    - php: 7.3
-      env: DB=mysqli
-    - php: nightly
-      env: DB=mysql
-    - php: nightly
-      env: DB=mysqli
 
   include:
-    - stage: Test
-      php: 7.1
-      env: DB=sqlite
-
-    - stage: Test
-      php: 7.2
-      env: DB=sqlite COVERAGE=yes
-
-    - stage: Test
-      php: 7.2
-      env: DB=mysql COVERAGE=yes
-
-    - stage: Test
-      php: 7.2
-      env: DB=mysqli COVERAGE=yes
-
     - stage: Test
       php: 7.1
       env: DB=mysql MYSQL_VERSION=5.7
       sudo: required
       before_script:
         - bash ./tests/travis/install-mysql-5.7.sh
     - stage: Test
-      php: 7.2
-      env: DB=mysql MYSQL_VERSION=5.7 COVERAGE=yes
+      php: 7.1
```

Comment:
7.1 was EOL'd today: @morozov what do we want to do about it, considered that the oatch targets `master`?

Reply:
Let's drop it in 2.10. All changes in 2.9 were done considering PHP 7.1, so we don't have to drop it immediately.